diff --git a/dist/shims/shims-workerd.js b/dist/shims/shims-workerd.js
index 8086c9a31186c553f9ad1f8e7aa912cdf413d89f..fed26fb30699bf1523c9abede51a734c8798d439 100644
--- a/dist/shims/shims-workerd.js
+++ b/dist/shims/shims-workerd.js
@@ -32,7 +32,7 @@ export function isTracingLoopRunningByDefault() {
     // request context. So we don't run the trace export loop by default.
     return false;
 }
-export { MCPServerStdio, MCPServerStreamableHttp } from "./mcp-server/browser.js";
+export { MCPServerStdio, MCPServerStreamableHttp, MCPServerSSE } from "./mcp-server/browser.js";
 export { clearTimeout, setTimeout } from 'node:timers';
 class NodeTimer {
     constructor() { }
diff --git a/dist/shims/shims-workerd.mjs b/dist/shims/shims-workerd.mjs
index 9931a7bc03b0f3817b31f5f5a4a6954e4c97e952..c872d9df827edc97be052fb5e1b2caa1e3e0acee 100644
--- a/dist/shims/shims-workerd.mjs
+++ b/dist/shims/shims-workerd.mjs
@@ -32,7 +32,7 @@ export function isTracingLoopRunningByDefault() {
     // request context. So we don't run the trace export loop by default.
     return false;
 }
-export { MCPServerStdio, MCPServerStreamableHttp } from "./mcp-server/browser.mjs";
+export { MCPServerStdio, MCPServerStreamableHttp, MCPServerSSE } from "./mcp-server/browser.mjs";
 export { clearTimeout, setTimeout } from 'node:timers';
 class NodeTimer {
     constructor() { }
